src/utils/airtableReturns.js

```diff
@@ -1,12 +1,12 @@
 const functions = {
-  returnEmptyArray: (res, err) => {
+  returnEmptyPayload: (res, err, value = []) => {
```

Comment:
Is this in case we want to send something back other than an empty array, even if we get nothing/an error from airtable?

Reply:
Yeah - `payload` for the /:station/case/:id API call is technically an empty object so I thought renaming made that more explicit and was more straightforward than creating `returnEmptyArray` and `returnEmptyObject` functions.